task/ecosystem-cert-preflight-checks/0.2/ecosystem-cert-preflight-checks.yaml

```diff
@@ -139,15 +141,22 @@ spec:
 
         # We purposely do not quote the array elements here, so that they are expanded by the shell as separate args.
         # shellcheck disable=SC2068
-        if ! retry skopeo inspect --retry-times "$skopeo_retries" ${_SKOPEO_INSPECT_ARGS[@]} "docker://${PARAM_IMAGE_URL}" \
-          | jq '.Labels | keys | .[]' -r \
-          | { grep operators.operatorframework.io.bundle || true ;} \
-          | tee /tmp/ecosystem-image-labels
+
+        # Saving skopeo output to a temp variable to be used later to create a file of labels and add digest back to the results
+        if ! _IMAGE_MANIFEST=$(retry skopeo inspect --retry-times "$skopeo_retries" ${_SKOPEO_INSPECT_ARGS[@]} "docker://${PARAM_IMAGE_URL}")
```

Comment:
This command will just get a singular image manifest so that the skopeo will succeed, matching either the platform that we are running on or the first manifest listed, right?

This means that if we have arm, ppc, and s390 architectures (but no x86), we will always inspect the arm architecture even if `params.platform` is set. Is this desired? If `params.platform` isn't set, do we assume that all labels are sufficiently consistent, or are we really only checking for the `operators.operatorframework.io.bundle` label, so this assumption should be reasonable enough? We actually have a separate requirement for all bundle images to be a simple image manifest, so we will likely not get into this case if that is what we are protecting against.